src/lambda/lambdaTreeDataProvider.ts

```diff
@@ -111,12 +127,20 @@ export class LambdaTreeDataProvider implements vscode.TreeDataProvider<AWSTreeNo
                 element: node,
                 outputChannel: this.lambdaOutputChannel,
                 resourceFetcher: this.resourceFetcher
-            })
+            }),
+            telemetryName: {
+                namespace: TelemetryNamespace.Lambda,
+                name: 'invoke_remote'
```

Comment:
If you use an underscore to join namespace and name, will we ever wish we used a different character within the name fields so that we could go back and perform reliable string splits on telemetry data?

Reply:
Thoughts, @hunterwerlla ?

Reply:
We can't use dots, so we need to use underscores. There is not reason we cannot split on underscores.

Reply:
@hunterwerlla parse on this arbitrary text into namespace (aws_credentials) and name (validate_mfa): `aws_credentials_validate_mfa`

Reply:
Solution is to remove underscores from everywhere else: the above metric would read `aws_credentialsvalidatemfa`

Reply:
✔️